.gitignore

```diff
@@ -7,4 +7,5 @@ prebuilds/
 lib/
 libexec/
 share/
+include/
```

Comment:
Ok, that's fine then. I was curious to know why would have not be included before on the `.gitignore` file, probably it's due to the changes on `BigRedButton` to earn some hard disk space after generated the prebuilds (SemaphoreCI only have about 2GB of space).